### ISSUE TITLE
network-manager: Disable mac address spoofing

### ISSUE THREAD

**Issue by kelebek333** [linuxmint/lmde-5-cinnamon-beta#36]
Hi,

Could you disable mac address spoofing for network manager? 

https://wiki.debian.org/WiFi/HowToUse#Troubleshooting_.26_Tips_for_NetworkManager
https://wiki.archlinux.org/title/NetworkManager#Configuring_MAC_address_randomization

Sample issues.

https://forums.linuxmint.com/viewtopic.php?p=1525481#p1525481
https://forums.linuxmint.com/viewtopic.php?p=1515858#p1515858

This setting comes by default at Ubuntu base.

```
cat /etc/NetworkManager/NetworkManager.conf 
[main]
plugins=ifupdown,keyfile

[ifupdown]
managed=false

[device]
wifi.scan-rand-mac-address=no


[keyfile]
```

**Comment by clefebvre** [comment]
Fixed in debian-system-adjustments 2022.03.09. Please close the issue if you're able to test the fix.

**Comment by kelebek333** [comment]
Thanks, it's working.
